client/src/components/BookGrid/BookGrid.jsx

```diff
@@ -8,11 +8,18 @@ import { useState } from "react"
 export default function BookGrid() {
     const [search, setSearch] = useState ("")
     const [results, setResults] = useState([])
+    const [fetching, setFetching] = useState(false)
 
     async function getSearch() {
         if (!search) return;
-        const response = await axios.get(`https://www.googleapis.com/books/v1/volumes?q=${search}&maxResults=35&langRestrict=en`)
-        setResults(response.data.items)
+        setFetching(true)
+        try {
+            const response = await axios.get(`https://www.googleapis.com/books/v1/volumes?q=${search}&maxResults=35&langRestrict=en`)
+            setResults(response.data.items)
+        } catch (err) {
+            alert("Couldn't retrieve search results.")
```

Comment:
Curious, in which case would this error fire?

Reply:
I put it there just in case the API is down or stops responding